src/main/java/io/cryostat/net/web/http/api/beta/HttpApiBetaModule.java

```diff
@@ -48,4 +48,29 @@
     @Binds
     @IntoSet
     abstract RequestHandler bindDiscoveryGetHandler(DiscoveryGetHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindProbeTemplateUploadHandler(ProbeTemplateUploadHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindProbeTemplateUploadBodyHandler(
+            ProbeTemplateUploadBodyHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindProbeTemplateDeleteHandler(ProbeTemplateDeleteHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindTargetProbePostHandler(TargetProbePostHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindTargetProbeDeleteHandler(TargetProbePostHandler handler);
+
+    @Binds
+    @IntoSet
+    abstract RequestHandler bindTargetProbesGetHandler(TargetProbePostHandler handler);
```

Comment:
This is re-binding the `TargetProbePostHandler` into the `RequestHandler` set. The corrected method definition should look like:

```java
    @Binds
    @IntoSet
    abstract RequestHandler bindTargetProbesGetHandler(TargetProbesGetHandler handler);
```

Note that the type of the argument has been updated to reflect the handler that was intended to be bound.

Reply:
This was a change that got lost since I worked on these changes in two separate branches, fixed it now.